metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1249,6 +1250,31 @@ class MetalsLanguageServer(
             val msg = s"Invalid arguments: $args. Expecting: $argExample"
             Future.failed(new IllegalArgumentException(msg)).asJavaObject
         }
+      case ServerCommands.NewScalaWorksheet() =>
```

Comment:
Also, I think we can start with new worksheet, but even create a `WizardProvider` and `NewCommands` that would generalize over creating templates.